lib/core/strings.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
   removeSpaces: (str) => {
     return str.trim().replace(/\s/g,'-');
+  },
+  removePathSpaces: (str) => {
```

Comment:
much better this name :100:

Reply:
I generally love to care about namings....